src/Form/countryNames.js

```diff
@@ -1,255 +1,562 @@
+    MM: 'Myanmar (Burma)',
+    MN: 'Mongolia',
+    MO: 'Macao',
+    MP: 'Northern Mariana Islands',
+    MQ: 'Martinique',
+    MR: 'Mauritania',
+    MS: 'Montserrat',
+    MT: 'Malta',
+    MU: 'Mauritius',
+    MV: 'Maldives',
+    MW: 'Malawi',
+    MX: 'Mexico',
+    MY: 'Malaysia',
+    MZ: 'Mozambique',
+    NA: 'Namibia',
+    NC: 'New Caledonia',
+    NE: 'Niger',
+    NF: 'Norfolk Island',
+    NG: 'Nigeria',
+    NH: 'Vanuatu',
+    NI: 'Nicaragua',
+    NL: 'Netherlands',
+    NO: 'Norway',
+    NP: 'Nepal',
+    NR: 'Nauru',
+    NU: 'Niue',
+    NZ: 'New Zealand',
+    OM: 'Oman',
+    PA: 'Panama',
+    PE: 'Peru',
+    PF: 'French Polynesia',
+    PG: 'Papua New Guinea',
+    PH: 'Philippines',
+    PK: 'Pakistan',
+    PL: 'Poland',
+    PM: 'St. Pierre & Miquelon',
+    PN: 'Pitcairn Islands',
+    PR: 'Puerto Rico',
+    PS: 'Palestinian Territories',
+    PT: 'Portugal',
+    PW: 'Palau',
+    PY: 'Paraguay',
+    QA: 'Qatar',
+    QO: 'Outlying Oceania',
+    RE: 'Réunion',
+    RH: 'Zimbabwe',
+    RO: 'Romania',
+    RS: 'Serbia',
+    RU: 'Russia',
+    RW: 'Rwanda',
+    SA: 'Saudi Arabia',
+    SB: 'Solomon Islands',
+    SC: 'Seychelles',
+    SD: 'Sudan',
+    SE: 'Sweden',
+    SG: 'Singapore',
+    SH: 'St. Helena',
+    SI: 'Slovenia',
+    SJ: 'Svalbard & Jan Mayen',
+    SK: 'Slovakia',
+    SL: 'Sierra Leone',
+    SM: 'San Marino',
+    SN: 'Senegal',
+    SO: 'Somalia',
+    SR: 'Suriname',
+    SS: 'South Sudan',
+    ST: 'São Tomé & Príncipe',
+    SU: 'Russia',
+    SV: 'El Salvador',
+    SX: 'Sint Maarten',
+    SY: 'Syria',
+    SZ: 'Eswatini',
+    TA: 'Tristan da Cunha',
+    TC: 'Turks & Caicos Islands',
+    TD: 'Chad',
+    TF: 'French Southern Territories',
+    TG: 'Togo',
+    TH: 'Thailand',
+    TJ: 'Tajikistan',
+    TK: 'Tokelau',
+    TL: 'Timor-Leste',
+    TM: 'Turkmenistan',
+    TN: 'Tunisia',
+    TO: 'Tonga',
+    TP: 'Timor-Leste',
+    TR: 'Turkey',
+    TT: 'Trinidad & Tobago',
+    TV: 'Tuvalu',
+    TW: 'Taiwan',
+    TZ: 'Tanzania',
+    UA: 'Ukraine',
+    UG: 'Uganda',
+    UK: 'United Kingdom',
+    UM: 'U.S. Outlying Islands',
+    UN: 'United Nations',
+    US: 'United States',
+    UY: 'Uruguay',
+    UZ: 'Uzbekistan',
+    VA: 'Vatican City',
+    VC: 'St. Vincent & Grenadines',
+    VD: 'Vietnam',
+    VE: 'Venezuela',
+    VG: 'British Virgin Islands',
+    VI: 'U.S. Virgin Islands',
+    VN: 'Vietnam',
+    VU: 'Vanuatu',
+    WF: 'Wallis & Futuna',
+    WS: 'Samoa',
+    XA: 'Pseudo-Accents',
+    XB: 'Pseudo-Bidi',
+    XK: 'Kosovo',
+    YD: 'Yemen',
+    YE: 'Yemen',
+    YT: 'Mayotte',
+    YU: 'Serbia',
+    ZA: 'South Africa',
+    ZM: 'Zambia',
+    ZR: 'Congo - Kinshasa',
+    ZW: 'Zimbabwe',
+    ZZ: 'Unknown Region'
+}
+
+/**
+ * Country names object using country name to reference 2-letter country codes
+ * Derived from the solution above with
+ * Object.fromEntries(Object.entries(COUNTRY_CODES_TO_NAMES).map(entry => [entry[1], entry[0]]))
```

Comment:
Initially I was generated it on the fly, but there's no harm in keeping this hardcoded. Saves computation on each page load.